main/guides/orchestration/getting-started/key-concepts.md

```diff
@@ -0,0 +1,235 @@
+        async seat => {
+          seat.exit();
+          const holder = await makeAccountKit();
+          return holder.asContinuingOffer();
+        },
+        'wantStakingAccount',
+      );
+    },
+  },
+);
+```
+
+
+
+### Zones
+Each [Zone](/glossary/#zone) provides an API that allows the allocation of [Exo objects](#exo) and Stores [(object collections)](https://github.com/Agoric/agoric-sdk/tree/master/packages/store/README.md) which use the same underlying persistence mechanism. This allows library code to be agnostic to whether its objects are backed purely by the JS heap (ephemeral), pageable out to disk (virtual), or can be revived after a vat upgrade (durable).
+
+
+See [SwingSet vat upgrade documentation](https://github.com/Agoric/agoric-sdk/tree/master/packages/SwingSet/docs/vat-upgrade.md) for more example use of the zone API.
+
+```javascript
+const zone = makeDurableZone(baggage);
+...
+zone.subZone('vows')
+```
+
+
+### ChainHub
+
+The `makeChainHub` utility manages the connections and metadata for various blockchain networks. It simplifies accessing and interacting with multiple chains, providing a unified interface for the orchestration logic to manage cross-chain operations effectively.
+
+```javascript
+const chainHub = makeChainHub(remotePowers.agoricNames);
+```
+
+# Recorder Kit Makers
+
+These are used to record state changes and events within the contract. `prepareRecorderKitMakers` is a utility that prepares the tools necessary to create these kits, ensuring that the contract can maintain a detailed log of its operations for auditing and debugging purposes.
+
+```javascript
+const { makeRecorderKit } = prepareRecorderKitMakers(baggage, marshaller);
+const recorderKit = makeRecorderKit(
+    privateArgs.storageNode,
+    /** @type {import('@agoric/zoe/src/contractSupport/recorder.js').TypedMatcher<MetricsNotification>} */ (
+      M.any()
+    ),
+)
+await recorderKit.recorder.write(proposalsArray);
+```
+
+### Vow Tools
+
+These tools handle promises and asynchronous operations within the contract. `prepareVowTools` prepares the necessary utilities to manage these asynchronous tasks, ensuring that the contract can handle complex workflows that involve waiting for events or responses from other chains.
+
+```javascript
+const vowTools = prepareVowTools(zone.subZone('vows'));
+...
+const makeLocalOrchestrationAccountKit = prepareLocalChainAccountKit(
+  zone,
+  makeRecorderKit,
+  zcf,
+  privateArgs.timerService,
+  vowTools,
+  makeChainHub(privateArgs.agoricNames),
+);
+...
+const makeCosmosOrchestrationAccount = prepareCosmosOrchestrationAccount(
+  zone,
+  makeRecorderKit,
+  vowTools,
+  zcf,
+);
+```
+
+
+### Durable Zone
+
+A zone specifically designed for durability, allowing the contract to persist its state across upgrades. This is critical for maintaining the continuity and reliability of the contract’s operations.
+```javascript
+const zone = makeDurableZone(baggage);
+```
+
+### Account Kits
```

Comment:
**Orchestration Account** is a key concept, represented as `OrchestrationAccountI` in [orchestration-api.ts](https://github.com/Agoric/agoric-sdk/blob/master/packages/orchestration/src/orchestration-api.ts).

"Kit" doesn't belong in the name of a key concept. An "X Kit" just means "An X and some related facets/stuff", as explained in the section on [Method Naming Structure](https://docs.agoric.com/guides/ertp/#method-naming-structure) _which is in an ERTP guide for no reason; it should probably go in the [JavaScript framework section](https://docs.agoric.com/guides/js-programming/)... probably under [separation of duties](https://docs.agoric.com/guides/js-programming/hardened-js.html#separation-of-duties)... and we should introduce the term **facet** near there_.